examples/Cargo.toml

```diff
@@ -7,7 +7,8 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.62"
-identity_iota = { path = "../identity_iota", features = ["memstore", "domain-linkage", "stronghold"] }
```

Comment:
Why the `stronghold` feature is removed, now the example doesn't work anymore. Not sure why CI didn't catch it, maybe because of `all-features` flag?

Reply:
Added stronghold crate to examples explicitly.